app/src/main/java/com/example/currencyconverter/data/network/dto/NetworkCurrency.kt

```diff
@@ -0,0 +1,26 @@
+package com.example.currencyconverter.data.network.dto
+
+import com.example.currencyconverter.data.database.entities.DatabaseCurrency
+
+data class NetworkCurrency(
+    val acronym : String,
+    val title : String
+)
+
+fun Map<String, String>.asNetworkData() : List<NetworkCurrency> {
```

Comment:
Kotlin allows to simplify syntax if your function returns value
```suggestion
fun Map<String, String>.asNetworkData() : List<NetworkCurrency>  =
map {
        NetworkCurrency(
            acronym = it.key,
            title = it.value
        )
    }
```